src/plugins/index_pattern_management/public/components/index_pattern_table/index_pattern_table.tsx

```diff
@@ -243,13 +246,22 @@ export const IndexPatternTable = ({ canSave, history }: Props) => {
     );
   })();
 
-  const description = ((
-    <FormattedMessage
-      id="indexPatternManagement.indexPatternTable.indexPatternExplanation"
-      defaultMessage="Create and manage the index patterns that help you retrieve your data from OpenSearch."
-    />
-  ) as unknown) as string;
-  const pageTitleAndDescription = showActionsInHeader ? (
+  const description = i18n.translate(
+    'indexPatternManagement.indexPatternTable.indexPatternExplanation',
+    currentWorkspace
+      ? {
+          defaultMessage:
+            'Create and manage the index patterns that help you retrieve your data from OpenSearch for {name} workspace. ',
```

Comment:
Nit: seems there is an extra space at the end?

Reply:
Thanks for catching it. Updated.